Qt: Minor text tweaks

8times9 opened the pull request:
Tweaks text largely in the graphics config descriptions but also some in the general config.

The idea is to have all of the option descriptions worded with an implied "This option" at the beginning, hence all of the changes to singular verb form.

BhaaLseN commented on Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp:
```diff
@@ -285,7 +285,7 @@ void EnhancementsWidget::AddDescriptions()
                  "visual quality, but also greatly increases GPU load and can cause issues in "
                  "certain games. Generally speaking, the lower the internal resolution is, the "
-                 "better your performance will be.\n\nIf unsure, select Native.");
+                 "better the performance will be.\n\nIf unsure, select Native.");
```
Is the second "the" even necessary here?

Tilka commented:
Please fix the lint issues. You will need to move text around a bit, not sure if there's a way to make clang-format do it automatically... Looks good otherwise.

8times9 commented:
@Tilka Done

8times9 commented:
Odd, I don't remember lint complaining about those last time. Fixed. 

I also found a typo in AdvancedWidget and fixed it.
`TR_TEXTURE_FORMAT_DECRIPTION --> TR_TEXTURE_FORMAT_DE**S**CRIPTION`

8times9 commented on Source/Core/DolphinQt/Settings/GameCubePane.cpp:
```diff
@@ -188,7 +188,7 @@ void GameCubePane::OnConfigPressed(int slot)
     bool ok;
     const auto new_mac = QInputDialog::getText(
-        this, tr("Broadband adapter MAC address"), tr("Enter new broadband adapter MAC address:"),
+        this, tr("Broadband Adapter MAC address"), tr("Enter new broadband adapter MAC address:"),
```
I capitalized that first one because it's a window title and we (mostly) capitalize every word in window titles. But because neither broadband nor adapter are proper nouns, in a sentence "broadband adapter" wouldn't be capitalized.

MayImilae replied:
Well, in this case it's probably best to capitalize it in both places, since what it is referring to in this case is the name of a piece of hardware that Dolphin is emulating - the GameCube Broadband Adapter.

8times9 replied:
Ohh, I didn’t know it was a specific piece of hardware! That makes more sense, will fix it.

8times9 commented on Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp:
```diff
@@ -335,14 +335,14 @@ void EnhancementsWidget::AddDescriptions()
       "stronger out-of-screen effects while a lower value is more comfortable.");
   static const char TR_3D_SWAP_EYES_DESCRIPTION[] =
-      QT_TR_NOOP("Swaps the left and right eye. Mostly useful if you want to view side-by-side "
+      QT_TR_NOOP("Swaps the left and right eye. Mostly useful to view side-by-side "
```
I feel like just "side-by-side mode" is a little ambiguous, what about "side-by-side rendering mode"?

8times9 commented:
@MayImilae I fixed your nitpicks

8times9 commented:
Sorry for all of the new changes, I decided to apply your input from #7196 

Let me know if there's anything else you'd like to see changed!

MayImilae commented on Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp:
```diff
@@ -285,76 +285,76 @@ void EnhancementsWidget::AddDescriptions()
   static const char TR_INTERNAL_RESOLUTION_DESCRIPTION[] =
       QT_TR_NOOP("Specifies the resolution used to render at. A high resolution greatly improves "
```
>Specifies the resolution used to render at.

I'm still surprised you didn't change this one! Might I suggest:

>Controls the rendering resolution of the emulated software.

8times9 replied:
Great idea, what do you think about just doing "Controls the rendering resolution" for conciseness?

MayImilae replied:
Well, I guess that's fine, but I wanted to make it clear what it is controlling. Up to you!

MayImilae commented on Source/Core/DolphinQt/Config/Graphics/AdvancedWidget.cpp:
```diff
@@ -145,72 +145,67 @@ void AdvancedWidget::OnEmulationStateChanged(bool running)
-      "the WASD keys to move the camera by a certain step distance (SHIFT+2 to move faster and "
-      "SHIFT+1 to move slower). Press SHIFT+R to reset the camera and SHIFT+F to reset the "
-      "speed.\n\nIf unsure, leave this unchecked.");
+      "Allows manipulation of the in-game camera. Move the mouse and hold the right button "
```
I'm missing the "while"s in this. "move and hold" is a little ambiguous, imo.

MayImilae replied:
Hm? You marked this as resolved, and thumbs uped it, without re-adding whiles? I'm confused, what did you adjust  here? Do you disagree?

8times9 replied:
Oh, I'm sorry! I agree, and I fixed it locally, but I held off on pushing any commits until you had answered the question about the rendering description. I guess that's confusing for other people, I'll do it differently in the future.

Sorry for the confusion!

8times9 commented:
@MayImilae Okay, I fixed those two issues

8times9 commented:
Sorry, I squeezed in one last commit to add a period haha. That should be everything.

stenzek commented:
@dolphin-emu-bot rebuild

MayImilae commented:
You'll need to fix that lint error before this can be merged.

8times9 commented:
Done

Helios747 commented:
@dolphin-emu-bot rebuild
